contracts/ClaimsManager.sol

```diff
@@ -373,14 +386,21 @@ contract ClaimsManager is
             "Too late to accept settlement"
         );
         claim.status = ClaimStatus.SettlementAccepted;
+        // If settlement amount in USD causes the policy coverage to be exceeded, clip the API3 amount being paid out
         uint256 settlementAmountInApi3 = claimIndexToProposedSettlementAmountInApi3[
                 claimIndex
             ];
-        emit AcceptedSettlement(claimIndex, claimant, settlementAmountInApi3);
-        IApi3Pool(api3Pool).payOutClaim(
-            claim.beneficiary,
-            settlementAmountInApi3
-        );
+        uint256 settlementAmountInUsd = claimIndexToProposedSettlementAmountInUsd[
+                claimIndex
+            ];
+        uint256 clippedAmountInUsd = updatePolicyCoverage(
+            claim.policyHash,
+            settlementAmountInUsd
+        );
+        uint256 clippedAmountInApi3 = (settlementAmountInApi3 *
+            clippedAmountInUsd) / settlementAmountInUsd;
```

Comment:
All this math is in the same unit (wei?), right?

Reply:
Yes, all amounts have 18 decimals